pytools/mpiwrap.py

```diff
@@ -13,7 +13,8 @@
 pytools.prefork.enable_prefork()
 
 
-if Is_initialized():  # noqa pylint:disable=undefined-variable
+# pylint:disable=undefined-variable
+if Is_initialized():    # type: ignore[name-defined,unused-ignore] # noqa
```

Comment:
I'm not getting the `named-defined` error locally, but it shows up on the CI, so I added the `unused-ignore`, which is not great :(